tools/tap-driver

```diff
@@ -166,11 +166,6 @@ class Driver:
 
         proc.wait()
 
-        # Make sure the test didn't change blocking output
-        assert fcntl.fcntl(0, fcntl.F_GETFL) & os.O_NONBLOCK == 0
```

Comment:
I have the feeling this will bite us again in the future.  Maybe github is just faster about emptying its pipes than semaphore was, and that's why it doesn't break anymore?

I'd like to find a more robust solution here...

If it works, though, I don't block[1] it.

[1] :roll_eyes: